### DISCUSSION TITLE
Recycling crate/container

### DISCUSSION THREAD

**Discussion by Hazmat616** [FakeFishGames/Barotrauma#12736] [Ideas and suggestions]
Have a single recycling crate capable of even holding ammo boxes and railgun shells be placed in subs whenever you dock and its purpose is to ease transporting things to be recycled such as ammo or welding fuel. Naturally it is removed when you leave. Alternatively designate a container and ammo shelves that can be linked to the outposts fabricator so we can more easily do this.

**Comment by SomeRandomNoobKekeke** [comment]
wait, if you can take it then you can steal it and use it whenever you like
idk about ammo boxes, but there is already[ a mod for railgun shells](https://steamcommunity.com/sharedfiles/filedetails/?id=2865067838)

**Reply by Hazmat616** [reply]
It gets removed when you undock so can't be stolen.

**Comment by bookslayer10** [comment]
Decent idea, but it seems a bit confusing that you would only have access to this on a station. Like, I get that it would be too powerful for normal submarine use so you can't have it during gameplay, but it doesn't make sense from the player's perspective.

I would rather just have a way to carry multiple heavy items at once all the time, and make players need to earn it.
